micronaut-langchain4j-core/build.gradle.kts

```diff
@@ -7,6 +7,10 @@ plugins {
 dependencies {
     api(mn.micronaut.context)
     api(libs.langchain4j)
+    implementation(platform(libs.langchain4j.community.bom))
+    compileOnly(libs.langchain4j.community.redis)
+    compileOnly(libs.langchain4j.community.neo4j)
+    compileOnly(libs.langchain4j.cassandra)
```

Comment:
there are already separate modules for neo4j and redis. I think these should be included in those modules.